cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -63,6 +68,7 @@
     private static final Long VOLUME_MOUNT_HACK_GROUPID = 1001L;
 
     public static final String METRICS_CONFIG_FILE = "config.yml";
```

Comment:
isn't it better having an explicit name around metrics i.e. `jmx-config.yml', 'metrics-config.yml' or something like that ?
If we are going to use the same ConfigMap for metrics and logging configuration maybe in the future we'll use the same ConfigMap for some other configs ... I don't have a use case for now but if it will happen I think that it's better having the configuration file name for metrics more explicit.